src/components/search/facets/selected-filters.vue

```diff
@@ -23,25 +23,24 @@
             :filter="filter"
           >
             <template #label="{ filter }">
-              <span>{{ filter.label }}</span>
-              <CrossTinyIcon />
+              <span class="x-text x-text--bold x-font-color--accent">{{ filter.label }}</span>
```

Comment:
Clear selected filters doesnt have the same height as the others

![image](https://user-images.githubusercontent.com/6597815/202393574-fc1b82ea-74a9-4088-be90-6beb5fbbd19d.png)

Reply:
Accepted by design, go ahead